openedx/adg/lms/applications/models.py

```diff
@@ -36,6 +45,14 @@ class ApplicationHub(TimeStampedModel):
         default=False, verbose_name=_('Written Application Submitted'),
     )
     submission_date = models.DateField(null=True, blank=True, verbose_name=_('Submission Date'), )
+    prerequisite_course_status = models.CharField(
+        verbose_name=_('Program Prerequisite Courses Status'), choices=STATUS_CHOICES,
+        max_length=15, default=NOT_STARTED,
+    )
```

Comment:
if a statement takes multiple lines, we prefer to split it with one argument per line. For example:
```
prerequisite_course_status = models.CharField(
        verbose_name=_('Program Prerequisite Courses Status'),
        choices=STATUS_CHOICES,
        max_length=15,
        default=NOT_STARTED,
)
```